<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFrame</class>
 <widget class="QFrame" name="MainFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1191</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Ustawienia Procesu</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Jakość</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Niska</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Średnia</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wysoka</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Poziom szczegółówości</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="DirectoryButton">
     <property name="text">
      <string>Wybierz ścieżkę folderu ze zdjęciami</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QTextEdit" name="textEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Podgląd obrazu 2D</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGraphicsView" name="graphicsView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Podgląd modelu 3D</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
